src/index.ts

```diff
@@ -428,37 +418,47 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
       eventBus.emit(Event.ERROR, event)
     }
 
-    eventSource.addEventListener('*', (msg: any) => {
-      const event: StreamEvent = JSON.parse(msg.data)
-
-      logDebug('Received event from stream: ', event)
-
+    const handleFlagEvent = (event: StreamEvent): void => {
       switch (event.event) {
         case 'create':
           setTimeout(() => fetchFlag(event.identifier), 1000) // Wait a bit before fetching evaluation due to https://harness.atlassian.net/browse/FFM-583
```

Comment:
It appears that the referenced ticket has been fixed, we might be able to remove the timeout